### PR TITLE
chore(starters): add gatsby-starter-flat-magazine

### PR DESCRIPTION
**Pull request by damnitrahul**
## Description
Add Gatsby Flat Magazine starter to Gatsby starter library

### CODE REVIEW

File: docs/starters.yml [hunk 1]
```diff
@@ -6959,3 +6959,25 @@
     - Uses SCSS for easy-to-understand naming
     - Google Analytics support
     - Responsive design
+- url: https://flat-magazine.netlify.app/
+  repo: https://github.com/damnitrahul/gatsby-starter-flat-magazine
+  description: A flat design Gatsby Starter Blog, Powered by Sanity to manage the content and styled using styled-components.
+  tags:
```
Comment (aaaakshat) [inline docs/starters.yml:6965]: Refer to [categories.yml](https://github.com/gatsbyjs/gatsby/blob/master/docs/categories.yml) to see the valid tags that can be added here. You cannot have tags like `Styled-components`, `Minimal Design` etc. as they aren't part of `categories.yml` if you believe they are missing you can add request additions to the file. However, we recommend using the available categories.

Comment (damnitrahul) [reply docs/starters.yml:6965]: Hey @aaaakshat  thank you for reviewing. I have updated the README and valid `tags`

**Comment by damnitrahul** [comment]
@sidharthachatterjee Can you please review.

**Comment by sidharthachatterjee** [comment]
@damnitrahul Just fixed conflicts. Please take a look at the Danger warning. The tag with index 5 is invalid.

**Comment by damnitrahul** [comment]
> @damnitrahul Just fixed conflicts. Please take a look at the Danger warning. The tag with index 5 is invalid.

Hey @sidharthachatterjee , It's caused by `CMS:sanity.io`, So do I remove the tag? or change something? The warning did not mention what do I need to do.

File: docs/starters.yml [hunk 2]
```diff
@@ -7393,6 +7393,25 @@
...
+    - Blog
+    - CMS:sanity.io
+    - SEO
+    - Search
+    - Pagination
+    - JavaScript
```
Comment (aaaakshat) [inline docs/starters.yml:7405]: JavaScript tag isn't valid for starters, only for sites categories in [`categories.yml`](https://github.com/gatsbyjs/gatsby/blob/master/docs/categories.yml)

Comment (damnitrahul) [reply docs/starters.yml]: Oops, Removed it!

**Comment by LekoArts** [comment]
Hi there! :wave: 

With the launch of the new, unified gatsbyjs.com comes some changes to the showcase and starter submission workflow. We want the showcase pages to be useful to the community members who visit them as well as those that submit contributions. That means reevaluating the submission processes and criteria. To allow more time to complete that work, we're pausing showcase submissions for the time being.

You can read more details in the PR here: #26442

I'm going to close this for now as the submission process likely won't be the same as we try to improve upon it, but please submit your work once we have updated it!

**Pull request closed without merging**